src/CommunicationLayer/CommunicationLayer.pro

```diff
@@ -1,10 +1,14 @@
 TEMPLATE = lib
-CONFIG += staticlib
+CONFIG += staticlib conan_basic_setup
 
 ! include(../common.pri){
     error("Could not find common.pri file!")
 }
 
+! include(../../build/conanbuildinfo.pri) {
```

Comment:
This will mess things up if we want to build from any directory other than `build`